git-extra/PKGBUILD

```diff
@@ -115,6 +115,6 @@ package() {
   install -m644 msys2-32.ico $pkgdir/usr/share/git
   install -m644 99-post-install-cleanup.post $pkgdir/etc/post-install
   install -m755 astextplain $pkgdir/usr/bin
-  install -m755 git-update $pkgdir${MINGW_PREFIX}/libexec/git-core
+  install -m755 git-upgrade $pkgdir${MINGW_PREFIX}/libexec/git-core
```

Comment:
That's not enough. You also need to update `install.iss`. And `make-file-list.sh`. Also, there are a couple of self-references in `git-update.sh` itself. (I used `git grep 'git.update[^-]` and `git grep 'git.exe.update[^-]'` to figure these out...)

Reply:
👍 sorry about that. I just looked at the initial commit so far.